### DISCUSSION TITLE
Sort Auth entries by usage frequency

### DISCUSSION THREAD

**Discussion by UnderTheCarpet** [ente-io/ente#988] [Enhancements]
It would be useful to have the most frequently used entries at the top. By adding an option to sort by usage frequency the probability to have the entry in need in close reach would likely rise while retaining a rather simple implementation.

**Comment by I-I-IT** [comment]
This would mean data collection though. Just allowing to modify the order should be enough.

**Comment by ua741** [comment]
This is available in latest version. Every information is still e2ee.

**Reply by I-I-IT** [reply]
Unfortunately this removed the pinned items

**Reply by I-I-IT** [reply]
Edit: it didn't remove but switch to weird custom order.
